### PR TITLE
Migrate structure board to iconfont

### PR DESCRIPTION
**Pull request by FinalAngel**
this also includes the submenu

### CODE REVIEW

**Comment by darioalbanesi** [comment]
lgtm

File: cms/static/cms/sass/components/_structureboard.scss [hunk 1]
```diff
@@ -49,7 +49,18 @@
         background: $color-grey-70;
     }
     .cms-dragarea-static {
-        background: $color-grey-70 url("../img/toolbar/pattern.png");
+        background-color: $color-grey-70;
+        background-image: linear-gradient(
+            135deg,
+            rgba(255, 255, 255, .1) 25%,
```
Comment (vxsx) [inline cms/static/cms/sass/components/_structureboard.scss]: we can do `rgba(white, 0.1)`

**Comment by FinalAngel** [comment]
yeah but we rather not use "white", "black" or in general color names. Thats even a lint setting in scss-lint

**Comment by vxsx** [comment]
yeah, but even `#fff` is better than `255, 255, 255`

**Comment by FinalAngel** [comment]
alright, let's fix that than on the next iteration, I'll add to jira

**Pull request merged**